firebase-inappmessaging/src/proto/firebase/inappmessaging/proto/experiment_payload.proto

```diff
@@ -13,9 +13,11 @@
 // limitations under the License.
 
 syntax = "proto3";
-package developers.mobile.abt;
 
-option java_outer_classname = "FirebaseAbt";
+package firebase.inappmessaging;
```

Comment:
Will there be any issues with serialization if we change the package name as I assume there could be a similar proto used at server side?

Reply:
nope, there should not be any issues

Reply:
Thanks for clarifying...I saw another proto from targeting inside FIAM maintaining the targeting package structure(.../firebase-inappmessaging/src/proto/developers/mobile/targeting/proto/client_signals.proto)...so was a little confused and added this question.

Reply:
Ah yeah I saw that but decided to keep it as-is since it was already in FIAM.